README.md

```diff
@@ -75,6 +75,7 @@ gauge run --log-level=debug --verbose --tags 'e2e & linux/amd64' specs/clusterta
 gauge run --log-level=debug --verbose --tags e2e specs/operator/rbac.spec
 gauge run --log-level=debug --verbose --tags e2e specs/operator/auto-prune.spec
 gauge run --log-level=debug --verbose --tags e2e specs/operator/addon.spec
+gauge run --log-level=debug --verbose --tags e2e specs/operator/pre-upgrade.spec:68
```

Comment:
Instead of this, could you write something like `gauge run --log-level=debug --verbose --tags e2e  <specification_path>:<scenario_line_number>` as it is given in the gauge [documentation](https://docs.gauge.org/execution?os=linux&language=python&ide=vscode)